### DISCUSSION TITLE
My first biped robot - Digit

### DISCUSSION THREAD

**Discussion by Chompalot** [makerforgetech/modular-biped#48] [Show and tell]
![P7010008](https://github.com/dmt-labs/modular-biped/assets/34891324/424dfc92-7ee2-4607-a34d-be5c9b0e1acf)

Built for the UK Robotic Games back in 2006, radio controlled, with 2 PIC micro controllers. One to decode the radio control and one to sequence the servos

**Comment by WhoIsMrSentry** [comment]
![V3](https://github.com/dmt-labs/modular-biped/assets/134494889/914b372a-afee-45e4-a363-ec560f306393)

It looks so cute @Chompalot  well done. it looks very nice 
My first robot is dan's companion-bot but with my own version

**Comment by danic85** [comment]
I love this! Are the feet copper board?

**Comment by Chompalot** [comment]
Hi Dan, sorry I missed this, the feet are just plywood, the legs are pcb like material, but just something I found in a skip.

**Comment by Chompalot** [comment]
This ia what I developed digit into for the Young Roboteers course we ran at Derby Museum. I chaged from PIC to Arduino Nano and no radio control just power on and a go button. The Young Roboteers got to do a little programming to adjust servo position, stride length and lean etc to get them walking as fast as possible and at the end of the course there was a race :)
![20230825_124938](https://github.com/dmt-labs/modular-biped/assets/34891324/df4a3a0d-fa06-4230-930b-0e204026e776)

**Reply by WhoIsMrSentry** [reply]
@Chompalot It's so cute 😀

**Comment by Chompalot** [comment]
After Digit I wanted to make a proper biped, so this was my effort.
I had access to a mlling machine so I was milling the joints ouf of HDPE
Again I was programming a PIC microconroller for sequences of pulses for the servos.
It only managed to stumble afer the initial servo positions. No gyros or feedback so it was shelved.

![20230825_131228](https://github.com/dmt-labs/modular-biped/assets/34891324/a8eb9a42-605a-4cb3-ab93-98c72d8ed8af)

**Comment by LMG26** [comment]
This was the beginning 

![image](https://github.com/dmt-labs/modular-biped/assets/14963180/f3c142d6-312a-4673-aea5-595f2cc36085)

**Reply by Chompalot** [reply]
That looks quite sturdy, did it ever get powered up?

**Reply by LMG26** [reply]
The servos we're not strong enough

**Comment by Chompalot** [comment]
This was one of my projects which sadly failed to work when powered up just now. The head should've been animated. I think it's an issue with the i2c pwm board (PCM9685) as I had issues with that before.

![20231028_141729](https://github.com/dmt-labs/modular-biped/assets/34891324/a620de9c-7ca6-428e-905b-d3a02404a755)
